schema.sql

```diff
@@ -0,0 +1,63 @@
+DROP DATABASE IF EXISTS books;
+
+CREATE DATABASE books;
+
+USE books;
+
+CREATE TABLE bookInfo (
+  id int NOT NULL AUTO_INCREMENT,
+  title VARCHAR(100) NOT NULL,
+  author VARCHAR(100) NOT NULL,
+  description TEXT NOT NULL,
+  PRIMARY KEY (id)
+);
+
+CREATE TABLE image (
```

Comment:
Do you need another table for book image? Is there more than one image for each book?

Reply:
Yeah, some of the books have more than one image and I want to show the counts in "more photo" like this: https://www.goodreads.com/book/show/15823480-anna-karenina